apps/pcq/pcq-consolidation-service/prod-00.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
 spec:
   values:
     job:
-      schedule: "0 18 1-31/2 * *"
+      schedule: "0 18 * * WED"
```

Comment:
Please update prod-01.yaml as well to run Thursday , just to be on the safe side.